streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -105,6 +105,8 @@
     private final TopicPartition t1p2 = new TopicPartition(topic1, 2);
     private final Set<TopicPartition> taskId02Partitions = mkSet(t1p2);
 
+    private final MockClientSupplier mockClientSupplier = new MockClientSupplier();
```

Comment:
This seems not used.

Reply:
Good catch -- missed to remove it after rebasing and moving the tests to `ActiveTaskCreatorTest`